.ci_support/linux_64_cdt_namecos6cuda_compiler_version11.0.yaml

```diff
@@ -3,7 +3,7 @@ c_compiler:
 c_compiler_version:
 - '7'
 cdt_name:
-- cos7
+- cos6
```

Comment:
This isn't right

Reply:
Hmm...this crept back in after the bot re-rendered. Must be missing something else still

Reply:
This was just because the re-rendering didn't pick up the new pinning change. Should be fixed now